spec/facades/foods_facade_spec.rb

```diff
@@ -0,0 +1,18 @@
+require 'rails_helper'
+
+RSpec.describe 'FoodsFacade', vcr: 'sweet_potatoes' do
+  it 'returns instances of food with given ingredient' do
```

Comment:
These are great tests. I would think about "Is there a way that we could get all the info we need - totalHits and the attributes we want with a single call?